_posts/2025-05-09-improve-opensearch-cluster-performance.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: "Improve OpenSearch cluster performance by separating search and indexing workloads"
+authors:
+  - naarcha
+  - pudyodu
+date: 2025-05-09
+categories:
+  - technical-posts
+meta_keywords: OpenSearch, shard roles, indexing, search separation, performance tuning
+meta_description: Learn how separating indexing and search workloads in OpenSearch can improve performance, scalability, and cost efficiency using dedicated roles and node types.
```

Comment:
Update meta with the following: 

_meta_keywords:_ OpenSearch cluster performance, index shards, search workloads, node roles, indexing and search separation, OpenSearch benchmark, performance tuning, optimizing your cluster

_meta_description:_  Optimize OpenSearch cluster performance by separating indexing and search operations. Discover write/search replicas and dedicated search nodes, with benchmark data showing 34% lower query latency through workload separation.